addon-test-support/@ember/test-helpers/dom/fire-event.js

```diff
@@ -180,8 +180,9 @@ function buildKeyboardEvent(type, options = {}) {
 }
 
 // eslint-disable-next-line require-jsdoc
-function buildFileEvent(type, element, files = []) {
+function buildFileEvent(type, element, options = {}) {
   let event = buildBasicEvent(type);
+  let files = options.files || [];
```

Comment:
As written, this is a breaking change. As mentioned in the PR description, anyone using `triggerEvent` for this today _must_ pass an array, if we are going to change this to `{ files: someArray }` as you suggest here, we definitely need to detect if `options` is already an array...

Reply:
> if we are going to change this to { files: someArray } as you suggest here, we definitely need to detect if options is already an array...

@rwjblue that makes sense. My first thought is to disambiguate this in the documentation itself, but then as a novice user of this add on, I would have expected that the interface to accept `{ files: someArray }`, hence why I felt it might be a more natural (but breaking change) step to standarize it.